Source/AGS.API/Objects/IHotspotComponent.cs

```diff
@@ -21,10 +21,10 @@ public interface IHotspotComponent : IComponent
 		PointF? WalkPoint { get; set; }
 
         /// <summary>
-        /// Gets or sets the hotspot text that can be showed by a hotspot label when hovering over the entity.
+        /// Gets or sets whether the hotspot name should be shown by a hotspot label when hovering over the entity.
```

Comment:
I would add to the comment: `<seealso cref="IEntity.DisplayName"/>`

Reply:
But it's in the next line. Or there's something else needed here?